README.md

```diff
@@ -140,14 +236,62 @@ Imagine that a deck of playing cards is sorted by rank and suit: first all the A
 
 * `rank(id)` returns 1-13, representing the card's rank (for an _id_ between 0-51).
 
```

Comment:
You're missing one vital part of this equation, but you're very close. Right now rank(0) is returning 0, instead of a number 1-13. Similarly, rank(51) is returning 12 instead of 13. Your ranks are just 1 number off...